src/locale/locales/de/messages.po

```diff
@@ -3511,45 +3373,45 @@ msgstr "Anzeigen"
 
 #: src/view/screens/PreferencesHomeFeed.tsx:68
 msgid "Show all replies"
-msgstr ""
+msgstr "Alle Antworten anzeigen"
 
 #: src/view/com/util/moderation/ScreenHider.tsx:132
 msgid "Show anyway"
 msgstr "Trotzdem anzeigen"
 
 #: src/view/com/modals/EmbedConsent.tsx:87
 msgid "Show embeds from {0}"
-msgstr ""
+msgstr "Eingebettete Medien von {0} anzeigen"
 
 #: src/view/com/profile/ProfileHeader.tsx:458
 msgid "Show follows similar to {0}"
-msgstr ""
+msgstr "Zeigen Sie Folgendes ähnlich wie {0}"
```

Comment:
Uh, tricky.

```suggestion
msgstr "Zeige ähnliche Follower an wie {0}"
```

Reply:
Yes this is tricky as I think the English wording here is a bit ambiguous. This string is the `accessibilityLabel` for the button next to the Follow button in profiles that shows you a list of similar accounts to follow. So I think given that 'follows' as a noun is difficult to translate, perhaps just translating `Show accounts similar to {0}`?

So would that be something like:
```suggestion
msgstr "Zeige ähnliche Konten wie {0}"
```